src/custom-element.js

```diff
@@ -282,6 +284,9 @@ export function createAmpElementProto(win, name, implementationClass) {
     /** @private {string|null|undefined} */
     this.mediaQuery_;
 
+    /** @private {!SizeList|null|undefined} */
```

Comment:
`{?SizeList|undefined}`, `{SizeList|undefined}` and `{!SizeList|null|undefined}` are all equivalent. `{SizeList|null|undefined}` is duplicative since it says `null` twice, but it's also equivalent. Generally, when we use `|undefined` we prefer to spell out all components instead of using shorthands.